src/services/wallets/operations/index.ts

```diff
@@ -1547,10 +1547,13 @@ export default class WalletOperations {
     const payloadTarget = signer.type;
     let isSigned = false;
 
-    if (
-      miniscriptSelectedSatisfier &&
-      (signer.type === SignerType.BITBOX02 || signer.type === SignerType.KEEPER)
-    ) {
+    const keysOnlyInSelectedPathSigners = [
+      SignerType.BITBOX02,
+      SignerType.KEEPER,
+      SignerType.POLICY_SERVER,
+      SignerType.INHERITANCEKEY,
```

Comment:
Keeper mobile key supports signing all paths it receives, why not have the server do the same?

Reply:
Will be fixing the childIndexArray out of sync issue later to establish consistency.